Packages/com.chisel.graph/Chisel/ChiselGraph.cs

```diff
@@ -0,0 +1,61 @@
+using Chisel.Core;
+using System.Collections.Generic;
+using UnityEngine;
+using XNode;
+
+namespace Chisel.Nodes
+{
+    [CreateAssetMenu(fileName = "New Chisel Graph", menuName = "Chisel Graph")]
+    public class ChiselGraph : NodeGraph
+    {
+        public ChiselGraphNode active;
+        public ChiselGraphInstance instance;
```

Comment:
But it seems that a graph is tied 1:1 to an instance? I guess this is a temporary thing to get started?

Reply:
Can have multiple instance using the same graph, but you are right, temporary